docs/articles/troubleshooting.md

```diff
@@ -15,6 +15,17 @@ If you don’t see any SFDX commands in the command palette, make sure that you
 
 See: [Java Configuration](getting-started/java-setup)
 
+## Set Salesforce CLI Path (Windows)
+
+See [Installing the Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli). If after installing the cli you see the error 'Salesforce CLI is not installed', this likely means that the Salesforce CLI was not added to your path. Follow these steps for verifying and adding the Salesforce CLI to your Windows path variable.
+
+1. From your [Windows Search](../images/troubleshooting_set_cli_path.1.png) look for `Edit the system environment variables`.
```

Comment:
I think the images should be added below the instructions. Right now they are links.

Reply:
Okay, I will adjust. I wasn't sure if that would be best since none of the other sections have images.